atef/widgets/config/window.py

```diff
@@ -72,24 +78,41 @@ def __init__(self, *args, show_welcome: bool = True, **kwargs):
         )
         self.action_print_report.triggered.connect(self.print_report)
         self.action_clear_results.triggered.connect(self.clear_results)
+
         if show_welcome:
             QTimer.singleShot(0, self.welcome_user)
 
     def welcome_user(self):
         """
         On open, ask the user what they'd like to do (new config? load?)
+
+        Set up the landing page
         """
-        welcome_box = QMessageBox(self)
-        welcome_box.setIcon(QMessageBox.Question)
-        welcome_box.setWindowTitle('Welcome')
-        welcome_box.setText('Welcome to atef config!')
-        welcome_box.setInformativeText('Please select a startup action')
-        open_button = welcome_box.addButton(QMessageBox.Open)
-        new_button = welcome_box.addButton('New', QMessageBox.AcceptRole)
-        welcome_box.addButton(QMessageBox.Close)
-        open_button.clicked.connect(self.open_file)
-        new_button.clicked.connect(self.new_file)
-        welcome_box.exec()
+        widget = LandingPage()
+        self.tab_widget.addTab(widget, 'Welcome to Atef!')
+        curr_idx = self.tab_widget.count() - 1
+        self.tab_widget.setCurrentIndex(curr_idx)
+
+        widget.new_passive_button.clicked.connect(
+            partial(self.new_file, checkout_type='passive')
+        )
+        widget.new_active_button.clicked.connect(
+            partial(self.new_file, checkout_type='active')
+        )
+        widget.sample_active_button.clicked.connect(partial(
+            self.open_file, filename=TEST_CONFIG_PATH / 'active_test.json'
+        ))
+
+        widget.sample_active_button.clicked.connect(partial(
```

Comment:
```suggestion
        widget.sample_passive_button.clicked.connect(partial(
```
Found through local testing

Reply:
I'm just keeping you on your toes 😉